stack.yaml

```diff
@@ -4,7 +4,7 @@
 #
 # NOTE: Remember to update the version matrix in README.md when
 # bumping the Cardano version.
-resolver: https://raw.githubusercontent.com/input-output-hk/cardano-haskell/c2aa1078ca8261d98cfce16d1131fbc687f6692b/snapshots/cardano-alonzo-purple-1.0.1.yaml
+resolver: tmp-snapshot.yaml
```

Comment:
If you would like to move the snapshot back into `stack.yaml`, feel free.

Reply:
I'm not sure if it helps the cache issues, but very happy to try. And even if not, it's easier to bump `cabal.project` and `stack.yaml` in the same repo than having the cardano-haskell indirection.

Reply:
Buildkite cache seems broken :(

Reply:
OK, pity - sounds like we need add a cache key then, based on the content of `stack.yaml`.